### PR TITLE
add schedule gotcha

### PR DESCRIPTION
**Pull request by zzstoatzz**
image

### CODE REVIEW

**Review by discdiver — CHANGES_REQUESTED** [review]
Thanks. Both of the deploy examples need `timedelta` imported, and a work pool name, and flow code source. Or maybe just make them `.serve`?

**Comment by zzstoatzz** [comment]
> Thanks. Both of the deploy examples need `timedelta` imported, and a work pool name, and flow code source. Or maybe just make them `.serve`?

good catches! updated in [2d4965b](https://github.com/PrefectHQ/prefect/pull/15803/commits/2d4965bdf4e232ae50aa2976ca7f352956f4b6ca)

File: docs/3.0/resources/upgrade-to-prefect-3.mdx [hunk 1]
```diff
@@ -259,4 +259,42 @@ async def my_flow():
 ```
 </CodeGroup>
 
-See the [Futures interface section](#futures-interface) for more information on this particular gotcha.
\ No newline at end of file
+See the [Futures interface section](#futures-interface) for more information on this particular gotcha.
+
+#### `TypeError: Flow.deploy() got an unexpected keyword argument 'schedule'`
```
Comment (zzstoatzz) [inline docs/3.0/resources/upgrade-to-prefect-3.mdx:264]:
```suggestion
#### `TypeError: Flow.serve() got an unexpected keyword argument 'schedule'`
```

**Review by discdiver — APPROVED** [review]
Cool. Just thought of this -  if you want to use the new mintlify highlighting syntax you can with {4} in the ```python     line (insert real line numbers).

**Pull request merged**